aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -98,10 +98,10 @@ For more information, see [Key storage providers](xref:security/data-protection/
 
 ## Deploy ASP.NET Core preview release to Azure App Service
 
-ASP.NET Core preview apps can be deployed to Azure App Service with the following approaches:
+Use one of the following approaches:
 
 * [Install the preview site extension](#install-the-preview-site-extension)
-<!-- * [Deploy the app self-contained](#deploy-the-app-self-contained) -->
+* [Deploy the app self-contained](#deploy-the-app-self-contained)
```

Comment:
Shouldn't this say "Deploy the self-contained app"?

Reply:
The app isn't a self-contained app when the dev is making this choice ... it would be in the default FDD state.

> Use one of the following approaches: ... Deploy the app self-contained

... as opposed to not deploying SCD and using one of the other options.